src/main.ts

```diff
@@ -3,7 +3,7 @@ import * as Tone from 'tone';
 
 class TestScene extends Phaser.Scene {
     synth: Tone.Synth
-
+// gaming
```

Comment:
```suggestion

    // gaming
```
example suggestion, I'd normally write something more detailed here